octoberry/octoberry/__main__.py

```diff
@@ -20,24 +26,33 @@
 handler.setFormatter(formatter)
 logger.addHandler(handler)
 
-def _play_sound(sound_url, cached_sound_path: Union[None, str] = None) -> None:
+
+def _play_sound(
+    sound_url: Union[None, str] = None, cached_sound_path: Union[None, str, pathlib.Path] = None
```

Comment:
I wish raspbian was shipped with python3.10+ so that we could use the better type hints syntax